DEVELOPERS.md

```diff
@@ -19,7 +19,7 @@ machine:
 
 * [Node.js v10.16.0 'Dubnium' (LTS)][node]: We use Node to generate the documentation, run a
   development web server, run tests, and generate distributable files. Depending on your system,
-  you can install Node either from source or as a pre-packaged bundle.
```

Comment:
"From source" is the usual wording (e.g., https://itsfoss.com/install-software-from-source-code/)